Themes/README.md

```diff
@@ -0,0 +1,14 @@
+
+# Contributing
```

Comment:
I think we should have some sort of description of what this extension is and maybe(?) a subset of your doc on colorization talking about how to override tokens.  Most theme extensions have a screenshot of the light/dark themes.  Maybe we can come up with a sample code snippet that shows off how this is better than existing themes.